guides/01_getting-started/01_quickstart.md

```diff
@@ -26,6 +26,16 @@ $code_hello_world
 
 $demo_hello_world
 
+## Local Development
+
+When developing locally, you can run Gradio apps from a Python script or a Jupyter Notebook. To run a Gradio app from a Python script, you can use the Gradio CLI to launch the application **in reload mode**, which will provide seamless and fast development.
```

Comment:
What are your thoughts on modifying Step 2 above slightly to read 

> Run the code below as a Python script (`python <filename>.py`) or in a Jupyter Notebook...

And then making this section you added strictly about reload mode? And we can link to the reload [guide ](https://gradio.app/developing-faster-with-reload-mode/) for more info

Reply:
Just because I think the part about running as a script is already mentioned in Step 2, so it would be better to clarify up there as opposed to another section.

Reply:
Added suggestions!